src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,104 @@
-import React from 'react';
+/* eslint-disable no-console */
+import React, { useState } from 'react';
 import './FindMovie.scss';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { getMovie } from '../../api/api';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  setMovieToList: (movie: Movie) => void,
+};
+
+export const FindMovie: React.FC<Props> = ({ setMovieToList }) => {
+  const [inputValue, setInputValue] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [checkFindMovie, setCheck] = useState(false);
+
+  const setTitleInput = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setInputValue(event.target.value);
+    setCheck(false);
+  };
+
+  const findMovie = async () => {
+    try {
+      const findMovieFromServer = await getMovie(inputValue);
+
+      if (findMovieFromServer.Response === 'True') {
+        setMovie(findMovieFromServer);
+      } else {
+        setMovie(null);
+        setCheck(true);
+      }
+    } catch {
+      throw new Error('Error');
+    }
+  };
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
+  const addMovieToList = (foundMovie: Movie) => {
+    if (foundMovie) {
+      setMovieToList(foundMovie);
+      setMovie(null);
+    }
+
+    setInputValue('');
+  };
+
+  return (
+    <>
+      <form className="find-movie">
+        <div className="field">
+          <label
+            className="label"
+            htmlFor="movie-title"
           >
-            Find a movie
-          </button>
+            Movie title
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className="input is-danger"
```

Comment:
```suggestion
              className="input"
```
клас is-danger має додаватись тільки при помилці (якщо не знайдений фільм). Зараз же він додається постійно (навіть коли нема помилки).
Для динамічного використання класів в реакті можна використовувати бібліотеку classnames
https://www.npmjs.com/package/classnames